source/docs/software/concepts/kinematics.rst

```diff
@@ -107,3 +107,38 @@ The inverse kinematics of a mecanum drive relate the desired velocity of the rob
    v_{br} = v_f - v_s + (2r_b \cdot \omega)
 
    v_{fr} = v_f + v_s + (2r_b \cdot \omega)
+
+Manipulators
+--------------
+
+.. _gravity-compensation:
+
+Gravity Compensation
+^^^^^^^^^^^^^^^^^^^^
+
+Often in FTC, teams have arms that swing out around an axis. Controlling these arms requires a bit of thought as depending on the angle they're at, the effect of gravity drastically changes.
+
+Our first step is defining a reference frame. Our recommendation is to define zero as straight up and down, as this is when gravity doesn't have an effect on your arm. Other reference frames will still work, but the trigonometry won't be as straightforward.
+
+.. figure:: ./images/kinematics/reference_frame.svg
+   :alt: A diagram of a robot with a long arm, with 0 degrees marked up and down
+   :align: center
+   :width: 400px
```

Comment:
```suggestion
```

Reply:
I don't know how this behaves on your computer, but on mine (firefox) implicitly sizing the image makes it super small. Thankfully, 400px is the intended width, and will appropriately resize to be smaller upon screen changes.

Reply:
I learned something about the HTML today: 
![image](https://github.com/gamemanual0/gm0/assets/16172089/61312abf-2fe3-4a64-a6b3-11f6b9d09d35)

We use a custom sphinx extension to set default image width/alignment, and it sets the sphinx image width attribute to 25 em, which works for PNGs/JPEGs as it gets set as the CSS width property, but for some reason on SVGs it gets set as the width tag on the `<img>` element, and so becomes 25 pixels wide. I have since fixed this, by making it set the image width attribute to 400 (as in, 400 pixels).